### ISSUE TITLE
Handles filenames with spaces incorrectly

### ISSUE THREAD

**Issue by clarkd** [squaredup/JarVerify#2]
Long filenames get truncated over multiple lines in the manifest, e.g.

```
Name: Perspectives/My very long filename goes here blah blah
  foobar whatever.json
SHA-256-Digest: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX=
```

If it happens to split the filename at a space, you end up with two spaces on the second line - as per the example above.

JarVerify would trim all this space, and attempt to verify a file called: `Perspectives/My very long filename goes here blah blahfoobar whatever.json`. Note the missing space between `blah` and `foobar`.

**Comment by clarkd** [comment]
This will be fixed in the next release.

**Comment by clarkd** [comment]
Fixed in 1.3.0

**Comment by clarkd** [comment]
NB: 1.3.0 is now available under our new organization on GitHub (https://www.nuget.org/packages/SquaredUp.JarVerify/1.3.0). You'll need to update your package reference to get the new version.
